release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -124,6 +124,13 @@ For more information, see xref:../installing/installing_azure/installing-azure-c
 
 This update introduces a new option for Ingress Controllers with the `hostnetwork` endpoint strategy. You can now host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports.
 
+[id=ocp-4-11-multiple-configuration-control-worker-nodes]
+==== Multi-node configuration for controller and worker nodes
```

Comment:
Usage of IPI is not recommended acc. to the OpenShift docs glossary here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#installer-provisioned-infrastructure

Reply:
Done